### DISCUSSION TITLE
SCSS file missing from build

### DISCUSSION THREAD

**Discussion by London804** [vercel/community#2983] [Bug Report]
### Page to Investigate

https://to-do-list-london804.vercel.app

### Version
"next": "13.4.4"

### Steps to Reproduce

I have a reset.scss file that removes styling on things like buttons that is not being applied when I build my app, but works perfectly fine locally. I'm using next.js and vercel for deployment. What's even more odd is that I can see some of my .scss files are loading correctly, but not the reset file. I don't see any errors in my logs. Please see the photos below. The first one is from a local dev version and the second is from a production build which can be found on the link above. 
Local:
<img width="1415" alt="image" src="https://github.com/vercel/community/assets/3270733/1779c7c4-971f-4bb5-b665-5cb16397ccf9">

Production:
<img width="1419" alt="image" src="https://github.com/vercel/community/assets/3270733/2444d8fe-e2ad-412f-8548-e4d934da75bd">

**Comment by amyegan** [comment]
Hi @London804. If you share your public repo or a minimal reproduction, someone here may be able to recognize what is going wrong 🙏

**Reply by London804** [reply]
Thanks @amyegan. The public repo can be found here. https://github.com/London804/to-do-list
